libdnf/dnf-context.cpp

```diff
@@ -225,6 +227,7 @@ static void
 dnf_context_init(DnfContext *context)
 {
     DnfContextPrivate *priv = GET_PRIVATE(context);
+    priv->vars_dir = g_strdup("/etc/dnf/vars");
```

Comment:
Maybe I'm misunderstanding something, but why are we forcibly setting this? We don't do this for `/etc/yum.repos.d` in this file?

Reply:
I think I put this in because I wanted it to have a default value. repo_dir gets its default value from elsewhere (https://github.com/rpm-software-management/libdnf/pull/622/files#diff-7038ad497716219392efbfc644c793d2R534). But maybe this isn't the right way to do it.

Reply:
I don't think you need this setting, as I think it's populated by the default value you've set already...